app/views/notFoundError.scala.html

```diff
@@ -0,0 +1,22 @@
+@*
+* This is a placeholder to allow us to write unit tests which show the redirect to the error page.
```

Comment:
Really nitpicky, but "redirect to the error page" should be "return an error page". Just in case anyone reads this and gets confused.

Reply:
You're right. I've changed it.